Step6 (#13)
JustineSara/proglang: ReadMe.md

to run the tests

```
clj -M -m main [file]
```

> run the `-main` function of `main` namespace. with no argument it starts a shell. With one argument, it reads a file and run the program in the file.

Reviewer: ```suggestion
run the `-main` function of `main` namespace. with no argument it starts a shell. With one argument, it reads a file and runs the program in the file.
```

**Part 2**: reworked the main so it handles the number of arguments and call the `run` or `run-file` functions. `run` is now not callable through `clj -X main/run` as it takes no input arguments now.

**Part 3**: completed the `run-file` function, checkink the file exists and then reading one line at a time and evaluating it. By default it only prints the last result. The options `-sao` and `-sai` shows all outputs and all inputs (and outputs).

### Step 6 : handle multiline program

**Goal**: handle multiline programs through the grammar.

> **Part 1**: modify the grammar and tree-making to add the new-line and disctinction between expression and statement (currently not there appart from the quitcommand).

Reviewer: ```suggestion
**Part 1**: modify the grammar and tree-making to add the newline and distinction between expression and statement (currently not there apart from the `quit!` command).
```

> **Part 2**: make it work for running a file: need to account for the last newline at the end of the file. Note that currently empty lines makes everything fail. Also removed the options as they are not relevant anymore.

Reviewer: ```suggestion
**Part 2**: make it work for running a file: need to account for the last newline at the end of the file. Note that currently empty lines make everything fail. Also removed the options as they are not relevant anymore.
```
